components/builder-api-client/src/error.rs

```diff
@@ -15,36 +15,71 @@
 use std::error;
 use std::fmt;
 use std::io;
+use std::num;
+use std::path::PathBuf;
 use std::result;
 
 use hyper;
 use serde_json;
 use url;
 
+use hab_core;
 use hab_http;
 
 pub type Result<T> = result::Result<T, Error>;
 
 #[derive(Debug)]
 pub enum Error {
     APIError(hyper::status::StatusCode, String),
+    BadResponseBody(io::Error),
```

Comment:
Yeah I _tried_ to do some reorganization where I saw things out of alphabetical order but i definitely missed a bunch. It would be great to clean all this up. Perhaps I'll whip up a follow on PR.